### DISCUSSION TITLE
Migration of data source with a "-" when migrating from 1.3.8

### DISCUSSION THREAD

**Discussion by hildeb** [phaag/nfsen#36] [Q&A]
I noticed that nfsen-1.3.9 (and more recent versions) enforces removing "-" (dash/minus) from Source identifiers.
But how can I migrate away from those while keeping the existing data.

Obviously we're currently still at 1.3.8 :)

**Comment by phaag** [comment]
You need to change the channel names in the respective info file in the stats directory and change all the flow files with something like this:

```
% find <channel> -type f -name nfcapd\* -exec nfdump -r {} -i "NewIdent" \;
```
